.gitignore

```diff
@@ -6,3 +6,4 @@
 /.tox
 /venv*
 .vscode
+.tox
```

Comment:
hmm that's weird, shouldn't line 6 have handled this?

Reply:
Hmm maybe I've overlooked something 🤔

Reply:
I can do Monday 😎but just pass the base URL to the API within the config.